README.md

```diff
@@ -33,15 +33,22 @@ their component variables.
 
 ## Installing
 
-It's recommended that you create a local python environment (with
-[Anaconda](https://www.anaconda.com/products/individual)). Otherwise, these tools can be installed with pip:
+It is currently recommended that you create a local python environment (with
+[Anaconda](https://www.anaconda.com/products/individual)) and install the
+sources as follows:
 
   ```shell
-  pip install google-weather-tools
+conda env create --name weather-tools --file=environment.yml
+conda activate weather-tools
+pip install .
+pip install weather_dl/
+pip install weather_mv/
+pip install weather_sp/
```

Comment:
Similar to the command above: the `pip install` command should be set in the `pip` section of the conda env.

Reply:
done